app/controllers/repos_controller.rb

```diff
@@ -138,4 +143,40 @@ def find_benchmark_type_by_category(category)
   def fetch_categories
     @repo.benchmark_types.pluck(:category)
   end
+
+  def generate_json
+    @charts.map do |chart|
+      # parse the Version string
+      ruby_versions = JSON.parse(chart[0][:categories]).map do |str|
+        str.split("<br>").reduce(Hash.new) do |memo, mapping|
```

Comment:
At a first glance, it seems wrong to be converting rendered data to a different rendered format -- that we should be generating this from the source data -- Unless the data gets stored in the database as HTML?